content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -224,16 +224,74 @@ merging, see
+| `//+listMapKey` | `x-kubernetes-list-map-keys` | List of field names, e.g. `["port", "protocol"]` | Only applicable when `+listType=map`. A list of field names whose values uniquely identify entries in the list. While there can be multiple keys, `listMapKey` is singular because keys need to be specified individually in the Go type. The key fields must be scalars. | 1.16 |
 | `//+mapType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to maps. `atomic` means that the map can only be entirely replaced by a single manager. `granular` means that the map supports separate managers updating individual fields. | 1.17 |
 | `//+structType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to structs; otherwise same usage and OpenAPI annotation as `//+mapType`.| 1.17 |
 
+If `listType` is missing, the apiserver will interpret a
+`patchMergeStrategy=merge` marker as a `listType=map` and will interpret
+the corresponding `patchMergeKey` marker as a `listMapKey`.
+
+The `atomic` list type is recursive.
+
+These markers are specified as comments and don't have to be repeated as
+field tags.
+
+#### Compatibility across topology changes
+
+On rare occurences, a CRD or built-in type author may want to change the
+specific topology of a field in one of their resource without bumping
+its version. Changing the topology of types, by upgrading the cluster or
+updating the CRD, will have different consequences when updating
+existing objects. There are two categories of changes: when a field goes
+from `map`/`set`/`granular` to `atomic` and the other way around.
+
+When the `listType`, `mapType`, or `structType` changes from
+`map`/`set`/`granular` to `atomic`, the whole list, map or struct of
+existing objects will end-up being owned by actors who owned an element
+of these types. This means that any further change to these objects
+would cause a conflict.
+
+When a list, map, or struct goes from `atomic` to `map`/`set`/`granular`,
+no conflict will be produced when objects with that structure is later
+changed. For that reason, it is not recommended to change a type from
+`atomic` to `map`/`set`/`granular`.
+
+Take for example, the custom resource:
+
+```yaml
+apiVersion: example.com/v1
+kind: Foo
+metadata:
+  name: foo-sample
+  managedFields:
+  - manager: manager-one
+    operation: Apply
+    apiVersion: example.com/v1
+    fields:
+      f:spec:
+        f:data: {}
+spec:
+  data:
+    key1: val1
+    key2: val2
+```
+
+Before `spec.data` gets changed from `atomic` to `granular`,
```

Comment:
Lines 286-288: Suggest to break up the sentence:
Before the `spec.data` field changes from `atomic` to `granular`,
`manager-one` owns the field `spec.data` and all objects within `spec.data` (`key1` and `key2`). 
A single manager owns an atomic [type] object completely?